docs/getting-started/uninstallation.md

```diff
@@ -13,6 +13,8 @@ rmdir -r -fo $env:LOCALAPPDATA\spicetify
 ```
 
 ## Linux and MacOS
```

Comment:
Maybe also add a note about if you used a package manager or something like AUR/flatpak/homebrew, to remove it the standard way for your package manager of choice.